src/ImageSharp/ColorSpaces/Conversion/ColorSpaceConverter.cs

```diff
@@ -2,100 +2,59 @@
 // Licensed under the Apache License, Version 2.0.
 
 using System.Numerics;
-using SixLabors.ImageSharp.ColorSpaces.Conversion.Implementation.LmsColorSapce;
-using SixLabors.ImageSharp.ColorSpaces.Conversion.Implementation.RgbColorSapce;
+using SixLabors.ImageSharp.ColorSpaces.Conversion.Implementation;
 
 namespace SixLabors.ImageSharp.ColorSpaces.Conversion
 {
     /// <summary>
-    /// Converts between color spaces ensuring that the color is adapted using chromatic adaptation.
+    /// Provides methods to allow the conversion of color values between different color spaces.
     /// </summary>
-    internal partial class ColorSpaceConverter
+    public partial class ColorSpaceConverter
     {
-        /// <summary>
-        /// The default whitepoint used for converting to CieLab
-        /// </summary>
-        public static readonly CieXyz DefaultWhitePoint = Illuminants.D65;
-
-        private Matrix4x4 transformationMatrix;
-
-        private CieXyzAndLmsConverter cachedCieXyzAndLmsConverter;
+        // Options.
+        private Matrix4x4 lmsAdaptationMatrix;
+        private CieXyz whitePoint;
```

Comment:
All white point members could be readonly!

Reply:
Thought I'd got these... Will update.